src/app/home/home.component.html

```diff
@@ -88,7 +88,7 @@ <h3 class="text-center">Reproducible environment</h3>
       <div fxLayout="column">
         <mat-icon fxFlexAlign="center" style="font-size: 50px; height: 50px">description</mat-icon>
         <h3 class="text-center">Describe tools and workflows</h3>
-        <p>Programs are described with <a href="https://www.commonwl.org/" target="_blank">CWL</a>, <a href="http://openwdl.org/" target="_blank">WDL</a> or <a href="https://www.nextflow.io/" target="_blank">Nextflow</a> and stored on <a href="https://github.com/" target="_blank">GitHub</a>, <a href="https://bitbucket.org/" target="_blank">BitBucket</a>, or <a href="https://www.gitlab.com" target="_blank">GitLab</a>.</p>
+        <p>Programs are described with <a href="https://www.commonwl.org/" target="_blank">CWL</a>, <a href="http://openwdl.org/" target="_blank">WDL</a> or <a href="https://www.nextflow.io/" target="_blank">Nextflow</a> and stored on <a href="https://github.com/" target="_blank">GitHub</a>, <a href="https://bitbucket.org/" target="_blank">Bitbucket</a>, or <a href="https://www.gitlab.com" target="_blank">GitLab</a>.</p>
```

Comment:
It was like this already, but since you're in there... You should add `rel="noopener noreferrer"` when using `target=_blank`.